From 49e23c25c73f771da1ab1506963be743f142a61a Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 10 Jun 2025 17:43:30 +0200
Subject: [PATCH] arm64: dts: st: fix lvds support on stm32mp255

Update clock-cells to introduce the lvds clock as argument into
device-tree board.

Change-Id: I18b91a19f6af2c227dc26a2df80d490abae0491a
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/468284
Tested-by: Jerome BESNARD <jerome.besnard@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Jerome BESNARD <jerome.besnard@st.com>
---
 arch/arm64/boot/dts/st/stm32mp255.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp255.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp255.dtsi
@@ -135,7 +135,7 @@
 	};
 
 	lvds: lvds@48060000 {
-		#clock-cells = <0>;
+		#clock-cells = <1>;
 		compatible = "st,stm32mp25-lvds";
 		reg = <0x48060000 0x2000>;
 		clocks = <&rcc CK_BUS_LVDS>, <&rcc CK_KER_LVDSPHY>, <&syscfg 0>;
